packages/omgidl-parser/src/parseIDLToAST.test.ts

```diff
@@ -1819,12 +1819,19 @@ module idl_parser {
     `;
     expect(() => parseIDLToAST(msgDef)).toThrow(/unexpected , token/i);
   });
-  it("cannot parse empty struct", () => {
+  it("can parse empty struct", () => {
```

Comment:
There's a comment on line 1801 that says that tests below this are not supported. Can you move this test above that?